src/pages/components/leadspace.mdx

```diff
@@ -41,26 +55,27 @@ This option has 4 empty columns on the left and right to accommodate pages that
 
 ### Lead space left-aligned
```

Comment:
Lead space left maybe? We aren't saying the other one is leadspace center-aligned.

Reply:
I believe it is "left-aligned", "right-aligned", "justified" and "centered". The common part is the suffix "-ed".  But I don't believe "center-aligned" would be too wrong either, just misleading, as nothing in it is aligned to the center, so considering its position as centered, I do suggest, though, sacrificing the "center-aligned" possibility as too questionable. So you can say the lead space is centered, but still, its elements are left-aligned.

I think this discussion should be included in the next component overhaul.

Reply:
I prefer "Lead space – Left" and "Lead space – Center". Note: Should we have the word "Lead space" in the naming?